sign/clin_sig/meta.yaml

```
@@ -0,0 +1,15 @@
+name: clin_sig
+type: "column consistent of strings, separated by ',' "
+description: |
+  This spell visualizes the clinical significance, given in clinvar significance terms
+link-to-url: https://www.ncbi.nlm.nih.gov/clinvar/
+example: 
+  code: |
+    render-table:
+      columns: 
+        Significance Column:
+          spell:
+            url: <url>
+testdata: data1.tsv
+authors:
+  -Benjamin Orlik
```

```suggestion
  - Benjamin Orlik
```

Still missing a whitespace here 😊